code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1094,24 +1096,24 @@ GLOBAL_LIST_EMPTY(PDAs)
 		to_chat(user, "Turn on your receiver in order to send messages.")
 		return
 
-	for (var/obj/item/pda/P in get_viewable_pdas())
-		if (P == src)
+	for (var/obj/item/pda/pda as anything in get_viewable_pdas())
+		if (pda == src)
 			continue
-		else if (P == aiPDA)
+		else if (pda == aiPDA)
 			continue
 
-		plist[avoid_assoc_duplicate_keys(P.owner, namecounts)] = P
+		plist[avoid_assoc_duplicate_keys(pda.owner, namecounts)] = pda
 
-	var/c = input(user, "Please select a PDA") as null|anything in sortList(plist)
+	var/choice = input(user, "Please select a PDA") as null|anything in sortList(plist)
 
-	if (!c)
+	if (!choice)
 		return
 
-	var/selected = plist[c]
+	var/selected = plist[choice]
 
 	if(aicamera.stored.len)
-		var/add_photo = input(user,"Do you want to attach a photo?","Photo","No") as null|anything in list("Yes","No")
-		if(add_photo=="Yes")
+		var/add_photo = input(user,"Do you want to attach a photo?","Photo","No") as null|anything in list("Yes","No") /// Change to TGUI List once ported
```

Comment:
No.

Reply:
(ike meant comment, we're not porting tgui lists)